config/config.go

```diff
@@ -25,21 +23,14 @@ func LoadConfiguration() *viper.Viper {
 	config.Token = v.GetString("CONSUL_TOKEN")
 
 	c, err := api.NewClient(config)
-	PanicOnError(err, "")
+	utils.PanicOnError(err, "")
 
 	pair, _, err := c.KV().Get(v.GetString("CONSUL_PATH"), nil)
-	PanicOnError(err, "Invalid consul config.")
+	utils.PanicOnError(err, "Invalid consul config.")
```

Comment:
If we are following go coding standard than error should be defined with lower case letter and no full stop at the end. 

Feel like utils.PanicOnError is an overkill as we can just do the if {blah.blah}